components/InputsOne.js

```diff
@@ -0,0 +1,64 @@
+import { useRef, useEffect, useState } from "react";
+import styled from "styled-components";
+
+export default function InputsOne({
```

Comment:
Same here for `InputsOne` as mentioned above. As this is the configuration part for your art generator maybe a more suitable as `InputsOne` can be found. :)

Reply:
Deleted "ONE"! 😩